spec/jobs/scheduled/destroy_past_events_spec.rb

```diff
@@ -48,6 +48,15 @@
     expect(Post.find_by(id: post.id)).not_to eq(nil)
   end
 
+  it "will destroy expired standalone events" do
+    topic = Fabricate(:topic)
+    event = CalendarEvent.create!(topic: calendar_post.topic, start_date: 10.years.ago)
```

Comment:
I'm surprise we have CalendarEvent here, this is a temporary artifact used after the migration from custom fields to tables but is not actually related to events, but only to topic calendars (let me know if that doesn't make sense).